README.md

```diff
@@ -16,7 +16,7 @@ all files together without compression, while having random access support.
 
 ### Install
 
-This module requires Node 10 or later.
+This module requires Node 22.12.0 or later.
```

Comment:
blocking: we should update the usage examples below to reflect the new non-default exports and use ESM syntax.